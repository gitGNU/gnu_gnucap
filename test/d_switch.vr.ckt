'
V1   1  0  dc 1. ac  1. 
R1   1  2  1.K
S1   2  0  3  0  sss 
V2   3  0  pwl( 0.  0.  5.  5.  15. -5.  25.  5. ) 
.model  sss  sw  ( vt= 0.  vh= 2.  ron= 1.K  roff= 1.E+12 )
.print dc v(nodes) ev(s1) 
.print tran v(nodes) ev(s1) 
.print ac v(nodes) ev(s1) 
.dc v2 -5 5 1 loop
.ac 1k
.tran 0 9 1
.tran
.tran
.ac 1k 
.list
.end 
