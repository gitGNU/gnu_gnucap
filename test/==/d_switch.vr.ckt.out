' 
#           v(1)       v(2)       v(3)       ev(S1)    
-5.         1.         1.        -5.         1.E+12    
-4.         1.         1.        -4.         1.E+12    
-3.         1.         1.        -3.         1.E+12    
-2.         1.         1.        -2.         1.E+12    
-1.         1.         1.        -1.         1.E+12    
 0.         1.         1.         0.         1.E+12    
 1.         1.         1.         1.         1.E+12    
 2.         1.         1.         2.         1.E+12    
 3.         1.         0.5        3.         1.K       
 4.         1.         0.5        4.         1.K       
 5.         1.         0.5        5.         1.K       
 5.         1.         0.5        5.         1.K       
 4.         1.         0.5        4.         1.K       
 3.         1.         0.5        3.         1.K       
 2.         1.         0.5        2.         1.K       
 1.         1.         0.5        1.         1.K       
 0.         1.         0.5        0.         1.K       
-1.         1.         0.5       -1.         1.K       
-2.         1.         0.5       -2.         1.K       
-3.         1.         1.        -3.         1.E+12    
-4.         1.         1.        -4.         1.E+12    
-5.         1.         1.        -5.         1.E+12    
#Freq       v(1)       v(2)       v(3)       ev(S1)    
 1.K        1.         1.         0.         1.E+12    
#Time       v(1)       v(2)       v(3)       ev(S1)    
 0.         1.         1.         0.         1.E+12    
 1.         1.         1.         1.         1.E+12    
 2.         1.         1.         2.         1.E+12    
 3.         1.         0.5        3.         1.K       
 4.         1.         0.5        4.         1.K       
 5.         1.         0.5        5.         1.K       
 6.         1.         0.5        4.         1.K       
 7.         1.         0.5        3.         1.K       
 8.         1.         0.5        2.         1.K       
 9.         1.         0.5        1.         1.K       
#Time       v(1)       v(2)       v(3)       ev(S1)    
 9.         1.         0.5        1.         1.K       
 10.        1.         0.5        0.         1.K       
 11.        1.         0.5       -1.         1.K       
 12.        1.         0.5       -2.         1.K       
 13.        1.         1.        -3.         1.E+12    
 14.        1.         1.        -4.         1.E+12    
 15.        1.         1.        -5.         1.E+12    
 16.        1.         1.        -4.         1.E+12    
 17.        1.         1.        -3.         1.E+12    
 18.        1.         1.        -2.         1.E+12    
#Time       v(1)       v(2)       v(3)       ev(S1)    
 18.        1.         1.        -2.         1.E+12    
 19.        1.         1.        -1.         1.E+12    
 20.        1.         1.         0.         1.E+12    
 21.        1.         1.         1.         1.E+12    
 22.        1.         1.         2.         1.E+12    
 23.        1.         0.5        3.         1.K       
 24.        1.         0.5        4.         1.K       
 25.        1.         0.5        5.         1.K       
 26.        1.         0.5        5.         1.K       
 27.        1.         0.5        5.         1.K       
#Freq       v(1)       v(2)       v(3)       ev(S1)    
 1.K        1.         0.5        0.         1.K       
V1  1  0  DC   1.  AC   1.
R1  1  2  1.K
S1  2  0  3  0   sss
V2  3  0  PWL(   0., 0.   5., 5.   15.,-5.   25., 5.)
.model sss sw (vt= 0.  vh= 2.  ron= 1.K  roff= 1.E+12)
.print dc v(nodes) ev(s1) 
.print tran v(nodes) ev(s1) 
.print ac v(nodes) ev(s1) 
.dc v2 -5 5 1 loop 
.ac 1k 
.tran 0 9 1 
.tran 
.tran 
.ac 1k 
